pkg/services/playlist/model.go

```diff
@@ -30,12 +29,9 @@ type PlaylistDTO struct {
 }
 
 type PlaylistItemDTO struct {
-	Id         int64  `json:"id"`
-	PlaylistId int64  `json:"playlistid"`
-	Type       string `json:"type"`
-	Title      string `json:"title"`
-	Value      string `json:"value"`
-	Order      int    `json:"order"`
+	Type  string `json:"type"`
+	Title string `json:"title,omitempty"` // ignored
+	Value string `json:"value"`
 }
 
 type PlaylistItem struct {
```

Comment:
don't we need `json` annotations in the `PlaylistItem` type now that we accept it as an input from the user?

Reply:
I don't think so -- rather it works fine without explicit `json:...` struct tags because they all use the default names anyway

Reply:
how is that possible? UpdatePlaylistCommand and CreatePlaylistCommand uses PlaylistItemDTO today for items which has lowercase json annotations. With this change wont the update/create http api require capitalized props for items?

Reply:
😬 -- actually not sure *how* it works, but just walked though all the APIs (that send lowercase) and they all work.

I'll go ahead and add explicit json tags too but not really sure why or how it works now

Reply:
`json.Unmarshal` prefers exact property name matches, but also accepts case-insensitive ones: 


> // To unmarshal JSON into a struct, Unmarshal matches incoming object
// keys to the keys used by Marshal (either the struct field name or its tag),
// preferring an exact match but also accepting a case-insensitive match. By
// default, object keys which don't have a corresponding struct field are
// ignored (see Decoder.DisallowUnknownFields for an alternative).


https://github.com/golang/go/blob/master/src/encoding/json/decode.go#L43-L47